Images referred in HTML templates are not found

##### **Overview of the issue**

I generated a project using jH 4.0.3 version. I copied all my actual navbar content into navbar.html. The logo in navbar.html is referring to `../../../content/images/my_logo.png`

All the images in default project are referred in css files and not from HTML files. I can't figure how to get this working with images referred directly from HTML. When I start the application, the image is not found. Also, I checked in www folder under build. There is no generated hash file name for `my_logo.png`.

##### **Motivation for or Use Case**

Component HTML must be able to refer to images in `.../content/images/...` directly without involving CSS

##### **Reproduce the error**

Add a new image in `/content/images` folder and refer to that image directly using `<img src> `

##### **JHipster Version(s)**

4.0.3

##### **Other**

node: v7.5.0
npm: 4.2.0
bower: 1.8.0
gulp: 3.9.1
yarn: 0.19.1

- [x] Checking this box is mandatory (this is just to show you read everything)

Based on http://stackoverflow.com/questions/35400733/webpack-and-angular-html-image-loading, I have to use html-loader instead of raw-loader. But for life of me, I can't figure why I'm getting lot of errors like below when I use html-loader

```
ERROR in ./src/main/webapp/app/account/password-reset/init/password-reset-init.component.ts
Module not found: Error: Can't resolve 'html-loader' in '/Users/company/folder/myapp'
 @ ./src/main/webapp/app/account/password-reset/init/password-reset-init.component.ts 46:18-65
 @ ./src/main/webapp/app/account/index.ts
 @ ./src/main/webapp/app/account/account.module.ts
 @ ./src/main/webapp/app/app.module.ts
 @ ./src/main/webapp/app/app.main.ts
 @ multi (webpack)-dev-server/client?http://localhost:9060 webpack/hot/dev-server ./src/main/webapp/app/app.main

```

All I'm doing is replacing raw-loader and index.html exclude with html-loader.

--- a/generators/client/templates/angular/_package.json
+++ b/generators/client/templates/angular/_package.json
@@ -55,6 +55,7 @@
     "extract-text-webpack-plugin": "2.0.0-beta.5",
     "file-loader": "0.10.0",
     "generator-jhipster": "<%= packagejs.version %>",
+    "html-loader": "0.4.5",
     "html-webpack-plugin": "2.28.0",
     "image-webpack-loader": "3.2.0",
     "jasmine-core": "2.5.2",
@@ -81,7 +82,6 @@
     "ts-node": "2.1.0",
     <%_ } _%>
     "proxy-middleware": "0.15.0",
-    "raw-loader": "0.5.1",
     "run-sequence": "1.2.2",
     "sourcemap-istanbul-instrumenter-loader": "0.2.0",
     "string-replace-webpack-plugin": "0.0.5",

--- a/generators/client/templates/angular/webpack/_webpack.common.js
+++ b/generators/client/templates/angular/webpack/_webpack.common.js
@@ -38,7 +38,7 @@ module.exports = function (options) {
                 },
                 {
                     test: /\.html$/,
-                    loader: 'raw-loader',
+                    loader: 'html-loader',
                     exclude: ['./src/main/webapp/index.html']
                 },
                 <%_ if (useSass) { _%>

